utils/Utils.js

```diff
@@ -91,6 +106,53 @@ class Utils {
       `⚠️ '${object}' has been deprecated. See https://github.com/OpenST/brandedtoken.js/issues/${issueNumber} for migration instructions.`,
     );
   }
+
+  /**
+   * Returns EIP712TypedData StakeRequest Object.
+   *
+   * @param stakeAmountInWei Stake amount in wei.
+   * @param btStakeRequestNonce BrandedToken StakeRequest nonce.
+   * @param brandedTokenAddress BrandedToken contract address.
+   * @param gatewayComposerAddress Gateway composer contract address.
+   *
+   * @returns {EIP712TypedData} EIP712TypedData instance of StakeRequest.
+   * @returns {EIP712Signature} Signature generated after EIP712 signing.
```

Comment:
The function returns EIP712TypedData and not EIP712Signature.
EIP712Signature is generated when doing Account signing.